src/plenoptic/metric/perceptual_distance.py

```diff
@@ -418,30 +447,41 @@ def nlpd(img1, img2):
 
     References
     ----------
-    .. [1] Laparra, V., Ballé, J., Berardino, A. and Simoncelli, E.P., 2016. Perceptual image quality
-       assessment using a normalized Laplacian pyramid. Electronic Imaging, 2016(16), pp.1-6.
+    .. [1] Laparra, V., Ballé, J., Berardino, A. and Simoncelli, E.P., 2016. Perceptual
+    image quality assessment using a normalized Laplacian pyramid. Electronic Imaging,
```

Comment:
this breaks sphinx's rendering of the reference. is it coming from one of the ruff rules?